### DISCUSSION TITLE
profile timezone

### DISCUSSION THREAD

**Discussion by tonmoycoder** [EddieHubCommunity/support#6171] [General] [answered]
![image](https://user-images.githubusercontent.com/78300713/207394184-4de3c880-44d5-4437-bb1f-d3a004518300.png)
Does anyone know how to do it? I can't figure out how to do it.

**Comment by Panquesito7** [comment, marked as answer]
This is a new feature recently added by GitHub :) and it's available in your profile settings.

1. Go to your [settings profile](https://github.com/settings/profile).
2. Scroll down a bit. You'll see a field `Display current local time`. Click on it and select your time zone.

![image](https://user-images.githubusercontent.com/51391473/208313631-74450d3e-59ce-4e76-ac6c-4a5d9cdfa8db.png)

4. Once you have your time set up, click on the green `Update profile` button.
5. Your timezone should be available in your profile now.

I hope that helped. If you need any more help, let us know! 🙂
